composer.json

```diff
@@ -50,7 +52,7 @@
     },
     "scripts": {
         "symfony-scripts": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
+            "cp .env.dist .env",
```

Comment:
could we remove `"incenteev/composer-parameter-handler"` from `"require"` then?

Reply:
Yes

Reply:
so... could you do it in this PR, please? 😅